htd/__init__.py

```diff
@@ -0,0 +1,283 @@
+import logging
+import socket
+import time
+from encodings import utf_8
+from typing import Callable
+
+import htd.utils
+
+from htd.constants import HtdConstants
+from htd.models import ZoneDetail
+
+_LOGGER = logging.getLogger(__name__)
+
+MAX_BYTES_TO_RECEIVE = 2 ** 10  # receive 1024 bytes
+ONE_SECOND = 1_000
+
+
+class HtdClient:
+    _ip_address: str = None
+    _port: int = None
+    _command_delay_sec: float = None
+    _retry_attempts: int = None
+    _socket_timeout_sec: float = None
```

Comment:
**nitpick:** I don't think you need this as they're all defined in your constructor.